core/client.go

```diff
@@ -14,9 +16,13 @@ type Client = client.Client
 
 // NewRemote creates a new Client that communicates with a remote Core endpoint over HTTP.
 func NewRemote(protocol, remoteAddr string) (Client, error) {
-	return http.New(
+	httpClient := retryhttp.NewClient()
+	httpClient.RetryMax = 2
```

Comment:
Why not 3? God loves Trinity

Reply:
3 actually means 4 tries :) bc it's 2 retries plus 1 try.